trunk/NDHMS/utils/module_version.F

```diff
@@ -0,0 +1,38 @@
+module module_version
+
+implicit none
+
+character (len=512), parameter :: wrfHydroVersion=WRF_HYDRO_VERSION
+character (len=512), parameter :: nwmVersion=NWM_VERSION
+
+private :: wrfHydroVersion, nwmVersion
+public  :: get_model_version, get_code_version, get_nwm_version`
+
+contains
+
+function get_model_version()
```

Comment:
this function should be removed?

Reply:
I think it's still useful for the "title" attribute on some output files.  I use it in module_NWM_io_dict.F.